### DISCUSSION TITLE
Kaggle機器學習競賽神器XGBoost介紹

### DISCUSSION THREAD

**Discussion by wangtyphoon** [1101-GS4719-C/Python-Community-GS4719-C#884] [優秀Python套件]
如果你想要更高的精準度和傑出的訓練速度，sklearn內有的模型已經不足以滿足你，極限梯度提升(xgboost)是你不可錯過的大好模型，詳情請見以下連結!
https://medium.com/jameslearningnote/%E8%B3%87%E6%96%99%E5%88%86%E6%9E%90-%E6%A9%9F%E5%99%A8%E5%AD%B8%E7%BF%92-%E7%AC%AC5-2%E8%AC%9B-kaggle%E6%A9%9F%E5%99%A8%E5%AD%B8%E7%BF%92%E7%AB%B6%E8%B3%BD%E7%A5%9E%E5%99%A8xgboost%E4%BB%8B%E7%B4%B9-1c8f55cffcc

**Comment by ching020721** [comment]
我曾使用過XGBoost這個model去做機器學習的訓練，誤差值跟一些model比起來真的進步不少，另外還有catBoost也是很不錯的模型，值得大家去嘗試！
